tsconfig.base.esm.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "module": "ES6",
+    "moduleResolution": "node",
+    "target": "es6",
+    "downlevelIteration": true
```

Comment:
`downlevelIteration` in ES6 code does not look right. `Symbol` should be supported for ES6. Also `downlevelIteration` produces more noise and less performant code.

Reply:
- [ ] disable `downlevelIteration`